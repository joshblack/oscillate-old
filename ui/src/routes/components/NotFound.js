import React from 'react';
import { Link } from 'react-router';

export default class NotFound extends React.Component {
  render() {
    return (
      <section>
        <p>Route not found</p>
        <Link to="/">Go back Home</Link>
      </section>
    );
  }
}
