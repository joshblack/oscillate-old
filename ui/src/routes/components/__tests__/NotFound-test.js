import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../NotFound';

describe('NotFound Component', () => {
  it('should render', () => {
    const tree = renderer.create(
      <NotFound />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
