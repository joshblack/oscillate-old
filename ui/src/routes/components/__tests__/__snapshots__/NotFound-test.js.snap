exports[`NotFound Component should render 1`] = `
<section>
  <p>
    Route not found
  </p>
  <a
    onClick={[Function]}
    style={Object {}}>
    Go back Home
  </a>
</section>
`;
